endoreg_db/models/patient/patient_examination.py

```
@@ -1,9 +1,17 @@
+from typing import TYPE_CHECKING, List
 
 # Serializer located in serializers/examination.py
 from typing import Optional
-from datetime import datetime
+if TYPE_CHECKING: 
+    from endoreg_db.models import (
+        Patient,
+        Examination,
+        Video,
+        PatientFinding,
+        PatientExaminationIndication,
```

Ruff (0.8.2)

`endoreg_db.models.PatientExaminationIndication` imported but unused

Remove unused import

(F401)